src/middlewares/request_huddles.js

```diff
@@ -50,18 +54,15 @@ function restructurePatients(patients) {
   return [];
 }
 
-export default function({ dispatch }) {
+export default function() {
   return next => action => {
-    switch (action.type) {
-      case FETCH_HUDDLES_FULFILLED:
-        let huddles = action.payload.data.entry;
-        dispatch({
-          type: FETCH_HUDDLES_RESOLVED,
-          payload: groupHuddles(huddles).map((huddleGroup) => restructureHuddles(huddleGroup))
-        });
-        return;
-    }
+    if (action.payload && action.payload.data && action.payload.data.Group) {
+      let huddles = action.payload.data.Group;
```

Comment:
I left it Group for consistency with other stuff so all of our keys are singular resource names.